src/Core/Components/List/FluentOption.razor.cs

```diff
@@ -45,6 +45,12 @@ public partial class FluentOption : FluentComponentBase
     [Parameter]
     public EventCallback<bool> SelectedChanged { get; set; }
 
+    /// <summary>
+    /// Gets or sets the content to display in the description slot.
+    /// </summary>
+    [Parameter]
+    public string? Description { get; set; }
```

Comment:
What is the difference between ChildContent and Description ? Maybe we can add this difference in the doc?

Reply:
See https://web-components.fluentui.dev/?path=/docs/components-dropdown-option--docs#start-content for the difference. Description is shown as an extra line under the ChildContent

Reply:
Probably better to have a 
`RenderFragment` property? And a name like `SubChildContent` or `DescriptionTemplate`.

Reply:
Then we would enable people to do things like this:

![image](https://github.com/user-attachments/assets/3884dcc6-821b-48a1-8ef6-24bc133026c1)

Think it is better to keep it a simple string. Think of it like the `Message` we have on the `FluentField`

Reply:
Make sense 😀. And the dev can use the `ChildContent` to format the content. 

Can you add an explanation in this property doc: to explain that is a message below the `ChildContent` content.